tests/integration/test_interaction.py

```diff
@@ -474,6 +541,15 @@ def test_item_negative_feedback_on_bad_input_with_keyboard(self, action_key):
     def test_final_feedback_and_reset_with_keyboard(self, action_key):
         self.parameterized_final_feedback_and_reset(self.items_map, self.feedback, action_key=action_key)
 
+    def test_alt_text_image(self):
+        self.alt_text_image_displayed(self.items_map)
```

Comment:
@dovinmu The `alt_text_image_displayed` method is not reused anywhere else, so you can just inline its body here, no need to put it in a separate method (same for the two texts below).